lib/docusign_transaction_rooms/models/meta_information.rb

```diff
@@ -0,0 +1,22 @@
+module DocusignTransactionRooms
+  class MetaInformation < BaseModel
+    include Kartograph::DSL
+
+    kartograph do
+      mapping MetaInformation
+      scoped :read do
+        property :resultSetSize
+        property :nextUri
+        property :previousUri
+        property :startPosition
+        property :endPosition
+      end
+    end
+
+    attribute :resultSetSize
```

Comment:
I'm not familiar with Kartograph, why are the attributes/properties duplicated.... seems like you could only need to configure the list once

Reply:
The Kartography bit is only really 5-14, these identify the JSON keys from which data is gathered to map onto the identified model....we can do cool things like rename keys but just didn't get around to it when I set up this gem.
To work correctly, the model just needs to be able to respond to similar_named attributes, for this gem, I use Virtus to create objects with attributes hence the appearance duplication.

Reply:
I guess you could define these attributes in a Constant and then loop over them for each section.  Not sure that would be quite as readable.  Meh.  I guess it's fine.